tests/coreos-install/positive/general.go

```diff
@@ -97,6 +96,30 @@ func init() {
 	})
 }
 
+// used by tests which want to test versions without pinning
+// a specific channel as on Container Linux machines the
+// channel will default to the host machines channel.
+//
+// will update the test.Version flag and replace the <Version>
+// flag on test.Name with the selected Version
```

Comment:
It doesn't do the latter.

Reply:
Whoops, leftover from an initial concept.